### PR TITLE
[Merge Ready]Adds in battle bridges, for when your bridge gets deleted

### LINKED ISSUE #640: Bridge Hit = ggnore
Right now, if the bridge gets hit, it's basically ggnore. If the hit doesn't outright kill the (usually unarmored, unsuited, no internals) bridge crew, it knocks them out and then there's nobody to either jump away or keep shooting.

I like the bit where the bridge gets hit and all the command crew die.
I don't like the bit where it knocks the ship out of the fight and gives the enemy free hits on us. That usually causes a lot of pain and can end the round.

I propose a sort of 'Battle Bridge' or secondary command post. A ship weapons console and a navigation console, placed somewhere other than the bridge. This would be one of the less-useful heads battle station. 

Possibly Engineering (use the engine control room?), possibly in the Brig (convert the HoS' office?), possibly Maint (above the bar hallway?). It wouldn't need to be a big room, or even a nice room. One chair, two consoles, and a box of donuts.

### PR DESCRIPTION
**Pull request by Kmc2000**
Fixes #640 

Adds in battle bridges, on the aetherwhisp, it replaces art storage, which is now in secure storage because that map is cramped...

On spaceSHIP.dmm it's just bolted onto the side

On the trailblazeit, it fills up some space. Unfortunately no room for a viewscreen

All of them come with their own shutters which are seperate.

:cl: Kmc2000 & Ike709
add: Added secondary bridges to the maps, on the aetherwhisp check near escapes. On the others it's very obvious
add: In these bridges, the main computers will now only work in general quaters (credit to Ike709)
/:cl:

### CODE REVIEW

**Comment by Kmc2000** [comment]
"turns out, having all your command staff and critical systems controls concentrated and exposed in a glass box is equivalent to fishing for angry sea beasties with your peen"

-Shuffl3

**Comment by yobob591** [comment]
Bridge hit is no longer game over 10/10

**Comment by monster860** [comment]
Awesome. Now we need to make it so you can control the various aspects of the ship directly from the things you are controlling

Like firing the weapons by using a ui on the weapon itself, FTLing by UI on FTL drive, shield toggling on the shields, etc.

**Comment by Kmc2000** [comment]
Adding in new stuff, dnm for now

**Comment by Vivalas** [comment]
Not merge ready, mapmerge and the conflicts will probably go away too.

**Comment by ike709** [comment]
>not using FastDMM with it's wonderful automatic map merging

File: code/game/machinery/computer/ftl_weapons.dm [hunk 1]
```diff
@@ -1,16 +1,23 @@
...
+	var/secondary = FALSE //For secondary Battle Bridge computers
+	var/general_quarters = FALSE //Secondary computers only work during General Quarters
 
 	var/datum/starship/target
 	var/datum/component/target_component
 
 /obj/machinery/computer/ftl_weapons/New()
 	..()
+	if(secondary)
+		desc = "This is a secondary Tactical computer, and older style model used only to supplement the main consoles when they become damaged. It will only work during General Quarters."
```
Comment (ike709) [inline code/game/machinery/computer/ftl_weapons.dm]: "and" should be "an"

File: code/modules/shuttle/ftl.dm [hunk 2]
```diff
@@ -38,9 +39,14 @@
...
 	var/do_send
 	var/icon_view_counter = 0
+	var/secondary = FALSE //For secondary Battle Bridge computers
+	var/general_quarters = FALSE //Secondary computers only work during General Quarters
+
 
 /obj/machinery/computer/ftl_navigation/New()
 	..()
+	if(secondary)
+		desc = "This is a secondary Navigation computer. It will only work during General Quarters."
```
Comment (ike709) [inline code/modules/shuttle/ftl.dm]: I like how you changed the secondary desc for the weapons computer but not the navigation computer.

Comment (ike709) [inline code/modules/shuttle/ftl.dm]: And didn't change the icon either.

File: code/modules/shuttle/ftl.dm [hunk 3]
```diff
@@ -38,9 +39,16 @@
...
 	var/do_send
 	var/icon_view_counter = 0
+	var/secondary = FALSE //For secondary Battle Bridge computers
+	var/general_quarters = FALSE //Secondary computers only work during General Quarters
+
 
 /obj/machinery/computer/ftl_navigation/New()
 	..()
+	if(secondary)
+		desc = "This is a Secondary Navigational computer, an older style model used only to supplement the main consoles when they become damaged. It will only work during General Quarters."
```
Comment (ike709) [inline code/modules/shuttle/ftl.dm]: Change "Secondary Navigational" to "secondary Navigation"

Comment (Vivalas) [inline code/modules/shuttle/ftl.dm]: They should probably both be lowercase.

File: code/game/machinery/computer/ftl_weapons.dm (hunk 1 above)
Comment (Vivalas) [inline code/game/machinery/computer/ftl_weapons.dm]: I would prefer if it was "tact

File: code/modules/shuttle/ftl.dm (hunk 3 above)
Comment (ike709) [inline code/modules/shuttle/ftl.dm]: Yeah probably but then he'd have to make Tactical lowercase too.

**Comment by ike709** [comment]
Just... Give me a minute...

**Comment by ike709** [comment]
https://github.com/FTL13/FTL13/pull/686

**Pull request merged**